pkg/services/policer/check.go

```diff
@@ -68,8 +69,13 @@ func (p *Policer) processNodes(ctx context.Context, addr *object.Address, nodes
 		}
 
 		if network.IsLocalAddress(p.localAddrSrc, node) {
-			shortage--
-		} else {
+			if shortage == 0 {
+				redundantLocalCopy = true
```

Comment:
Can't we call `p.cbRedundantCopy(addr)` right there? The shortage is already `0` anyway.

Reply:
I thought that callback will be deeply nested in for-loop and if-statement, what do u think @alexvanin?

Reply:
I didn't notice any issues with callback readability inside the loop, however I don't mind to have it explicitly outside. Just wonder maybe there are other reasons to put callback outside of loop.

Reply:
I'll leave the comment that we can slightly improve the performance instead of readability, so we can return to this case later when this place of code will become some kind of "stable".